config/routes.rb

```diff
@@ -1,3 +1,9 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  resources :questions
+
+  resources :choices, only: [:create, :destroy]
```

Comment:
FYI, if choices belong to questions then choice resource should be a child of questions. and then your path will be look like
POST /questions/: question_id/choices/
DELET  /questions/: question_id/choices/:id

Reply:
updated